scripts/teamDataScript.js

```diff
@@ -232,55 +232,71 @@ function savePitAnswers() //onclick for save pit button
     
     $('#pitTableConfirmation').text('Answers saved.');
     showAllPitDataTable();
-}
+}   
 
 function showAllPitDataTable() //shows pit stats of all teams for the current event in one table
 {
     var season = getCurrSeason();
     var event = getCurrEvent();
     var cname = season + ' pitQuestions';
-    var questionsArr = getCookie(cname).split('Ω').slice(0,-1); //first element to second to last element of .split arr
-    $('#allTeamsPitAnswers').html('');
+    var questionsArr = getCookie(cname).split(COOKIE_QUESTION_SEPARATOR).slice(0,-1); //first element to second to last element of .split arr
+    
+    var initialTableHTML = '<tr class="teamRow"><th class="allPitTableHeader">Team #</th></tr>';
+    $('#allTeamsPitAnswers').html(initialTableHTML);
     $('#allPitAnswersTitle').text(`All Pit Scouting Data for the ${event} event.`);
 
-    var tableHeader = '<tr><th class="allTeamsRow">Team #</th>'; //top-left most cell
-    for (var i = 0; i < questionsArr.length; i++)
+    //adding initial rows with question headers
+    for (var i = 0; i < questionsArr.length; i++) //for each question
     {
-
-        tableHeader += `<th>${questionsArr[i]}</th>`;
+        var questionWithNoSpaces = stripAlphaNumerics(questionsArr[i]);
+        var questionHeaderHTML = `<th class="allPitTableHeader">${questionsArr[i]}</th>`;
+        var newRowHTML = `<tr class='${questionWithNoSpaces}'>${questionHeaderHTML}</tr>`; //the question is one of the classes (w/o whitespaces), will help with adding team data
```

Comment:
this didn't occur to me before, but we should do a few things here:
* sanitize html like code (to prevent what was meant as text from getting rendered)
* replace carriage-return and line feed combinations with `<br/>`  (so that blank lines in the notes stay as blank lines - if someone took the time to put a separator in for readability, we should respect it).

For sanitizing (and do this before you do the new lines), just replace all `<` characters with `&lt;` (so they render as a `<` but don't get interpreted as html code). (`...>${questionHeaderHTML.replace('<', '&lt;')}</tr>`)

For replacing new lines/carriage-returns, add `.replace(/(\r\n|\n|\r)/gm, "<br/>")`
(the `/(\r\n|\n|\r)/gm,` is a regular expression replacement that will match Windows, Linux/Android, and Mac line seperators).

The end effect should be something like:
```js
...>${questionHeaderHTML.replace('<', '&lt;').replace(/(\r\n|\n|\r)/gm, "<br/>")}</tr>
```
(although, you could make it a function).

Reply:
('nother though on sanitation) since ampersands are used as html to signify a character, they will get "dropped" if they're typed in (and not a valid unicode value like `&lt;` or `&1F642;`, so replace them to (`.replace('&', '&amp;')`) (make sure to do this first so that you don't turn your less than's into ampersand-lt's).